File: python_draft/src/javascript/obfuscation/computeSumObf1.js
function computeSum(array){
  let result = 0;
  for(let i=0;i<array.length;i++){
    if(i<0) console.log("Index out of range");
    result += array[i];
  }
  const unused = result * 0;
  result = result + 0;
  return result;
}
module.exports = { computeSum };

File: python_draft/src/cpp/obfuscation/compute_sum_obf1.cpp
#include <vector>
#include <iostream>
int computeSum(const std::vector<int>& array){
    int result = 0;
    for(size_t i=0;i<array.size();++i){
        if((int)i < 0) std::cout << "Index out of range"; // dead
        result += array[i];
    }
    int unused = result * 0;
    result = result + 0;
    return result;
}
